### ISSUE TITLE
Just running init fails

### ISSUE THREAD

**Issue by hannojg** [muxinc/cli#22]
Tried to find a way to quickly check out mux (beyond the demo on the website). Thought the cli might be a way.

Installed with npm globally, running `mux init myenv.env` brings up this error:

```    
Error: Cannot find module '/usr/local/lib/node_modules/@mux/cli/lib/commands/init'
    Require stack:
    - /usr/local/lib/node_modules/@mux/cli/node_modules/@oclif/config/lib/plugin.js
    - /usr/local/lib/node_modules/@mux/cli/node_modules/@oclif/config/lib/config.js
    - /usr/local/lib/node_modules/@mux/cli/node_modules/@oclif/config/lib/index.js
    - /usr/local/lib/node_modules/@mux/cli/node_modules/@oclif/command/lib/command.js
    - /usr/local/lib/node_modules/@mux/cli/node_modules/@oclif/command/lib/index.js
    - /usr/local/lib/node_modules/@mux/cli/bin/run
    Code: MODULE_NOT_FOUND
```

as I was just about to quickly checkout mux I am not diving into debugging this on my own, maybe you can resolve this issue on your side?

@mux/cli/0.6.0 darwin-x64 node-v12.19.0
(also tried with node 14, doesn't work)

Thanks in advance

**Comment by roger-o3h** [comment]
Same issue...

**Comment by eropple** [comment]
Hey there! I apologize for the lateness of my reply; it looks like we didn't have mux-cli surfacing notifications in our Slack. We've fixed that this morning.

We've got a fix in progress and it should be released to NPM after review today.

**Comment by eropple** [comment]
Fixed in #23. Thank you for the report!
